### ISSUE TITLE
synapse_port_db: RuntimeWarning: coroutine 'BackgroundUpdater.do_next_background_update' was never awaited

### ISSUE THREAD

**Issue by babolivier** [matrix-org/synapse#6719]
Running `synapse_port_db` results in:

```
RuntimeWarning: coroutine 'BackgroundUpdater.do_next_background_update' was never awaited
```

being thrown in the log and the script wedging itself. This was introduced in #6647.

**Comment by richvdh** [comment]
I think this was maybe fixed by #6718?

**Comment by babolivier** [comment]
Whoops, indeed.
